core/library.py

```diff
@@ -157,7 +158,7 @@ def get_movies(url):
                 else:
                     movie['resolution'] = 'DVD-SD'
             else:
-                movie['videocodec'] = movie['resolution'] = ''
+                movie['videocodec'] = ''
```

Comment:
The resolution key needs to be assigned somewhere or it will be null in the database (None in python) and will raise exceptions any time it is compared to other strings.